src/web_performance/serving_pre-compressed_content.conf

```diff
@@ -0,0 +1,57 @@
+# ----------------------------------------------------------------------
+# | Serving pre-compressed content                                     |
+# ----------------------------------------------------------------------
+
+<IfModule mod_headers.c>
+
+    # Serve brotli compressed CSS, JS, HTML, SVG, ICS and JSON files if they exist 
+    # and the client accepts br.
+    RewriteCond "%{HTTP:Accept-encoding}" "br"
+    RewriteCond "%{REQUEST_FILENAME}\.br" -s
```

Comment:
I would use the `-f` flag ([see docs](https://httpd.apache.org/docs/current/mod/mod_rewrite.html#rewritecond))